autogluon/utils/tabular/features/generators/auto_ml_pipeline.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from .pipeline import PipelineFeatureGenerator
+from .category import CategoryFeatureGenerator
+from .datetime import DatetimeFeatureGenerator
+from .identity import IdentityFeatureGenerator
+from .text_ngram import TextNgramFeatureGenerator
+from .text_special import TextSpecialFeatureGenerator
+from ..feature_metadata import R_INT, R_FLOAT
+
+logger = logging.getLogger(__name__)
+
+
+class AutoMLPipelineFeatureGenerator(PipelineFeatureGenerator):
+    """
+    Pipeline feature generator with simplified arguments to handle most Tabular data including text and dates adequately.
+    This is the default feature generation pipeline used by AutoGluon when unspecified.
+    For more customization options, refer to PipelineFeatureGenerator and BulkFeatureGenerator.
+
```

Comment:
Add TODO: write out in English the full set of transformations that are applied (and eventually host page on website).  Also explicitly write out all of the feature-generator "hyperparameters" that might affect the results from the AutoML FeatureGenerator

Reply:
added